### PR TITLE
Config file for Artillery genius pro

### PR DESCRIPTION
**Pull request by CleisonManriqueAguirre**
I recently created this config file for those who have the Artillery genius pro

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "cmake.sourceDirectory": "${workspaceFolder}/lib/rp2040/boot_stage2"
```
Comment (JamesH1978) [inline .vscode/settings.json:2]: Can i ask why this is needed?

File: config/example_artillery_genius_pro_2023.cfg [hunk 2]
```diff
@@ -0,0 +1,153 @@
+[include mainsail.cfg]
```
Comment (JamesH1978) [inline config/example_artillery_genius_pro_2023.cfg:1]: Please do not put custom includes in stock files, the end user may not have mainsail

File: config/example_artillery_genius_pro_2023.cfg [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+pin: PC8
+off_below: 0.1
+[heater_fan extruder]
+pin: PC7
+off_below: 0.2
+[controller_fan case]
+pin: PC6
+off_below: 0.3
+idle_speed: 0.0
+[temperature_sensor mainboard]
```
Comment (JamesH1978) [inline config/example_artillery_genius_pro_2023.cfg:78]: Please do not add custom temperature sensors for mcu's and hosts, these may not be available on the end users equipment

File: config/example_artillery_genius_pro_2023.cfg [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+screw1_name: front left
+screw2: 143,63
+screw2_name: front right
+screw3: 143,183
+screw3_name: back right
+screw4: 23,183
+screw4_name: back left
+speed: 100.0
+screw_thread: CW-M5
+[mcu]
```
Comment (JamesH1978) [inline config/example_artillery_genius_pro_2023.cfg:124]: You have stated [mcu] twice

**Review by JamesH1978 — CHANGES_REQUESTED** [review]
Good afternoon,

Thank you for submitting this config to the repo, I have a few comments. You will need to add a signed offline as per https://github.com/Klipper3d/klipper/blob/master/docs/CONTRIBUTING.md

And you also need to add regression tests as per point 4 in https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md and review other needs in there too.

Thanks
James

File: config/example_artillery_genius_pro_2023.cfg (hunk 3 above)
Comment (CleisonManriqueAguirre) [reply config/example_artillery_genius_pro_2023.cfg:78]: Hi, they does work perfectly on Artillery genius .

File: config/example_artillery_genius_pro_2023.cfg (hunk 4 above)
Comment (CleisonManriqueAguirre) [reply config/example_artillery_genius_pro_2023.cfg:124]: Yes , I will solve it

**Review by JamesH1978 — COMMENTED** [review]
Please refer to my original comment, you need to:

You will need to add a signed offline as per https://github.com/Klipper3d/klipper/blob/master/docs/CONTRIBUTING.md

You also need to add regression tests as per point 4 in https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md and review other needs in there too. Please make the file look the same as other configs, and include flash instructions at the top of the file. Also point 1v this would not be an example but a printer.

Thanks
James

**Pull request closed without merging**